src/Orchard.Web/Modules/Orchard.Layouts/Assets/JavaScript/LayoutEditor/Services/ScopeConfigurator.js

```diff
@@ -281,6 +284,10 @@
                                         element.setIsDropTarget(false);
                                         if (!!receivedElement)
```

Comment:
:D

Reply:
Igen ez nekem is feltünt dev-be be is másoltam. I can't even.